src/components/CartAside/CartAside.tsx

```diff
@@ -1,50 +1,130 @@
 import './CartAside.scss';
-import React from 'react';
 import { AiOutlineClose } from 'react-icons/ai';
 import { Link } from 'react-router-dom';
+import { RootState } from '../../store/store';
+import { closeCart } from '../header/headerAsideSlice';
+import { useDispatch, useSelector } from 'react-redux';
+import React from 'react';
 import bowl from '../../assets/images/bowl.jpg';
+import rightArrow from '../../assets/images/rightArrow.svg';
+import shoppingbag from '../../assets/images/shoppingbagx.svg';
 
 export default function CartAside() {
+  const asideOpener = useSelector((state: RootState) => state.asideOpener);
+  const dispatch = useDispatch();
+
+  const closeCartAside = () => dispatch(closeCart());
+
   return (
-    <aside className="cart-aside-field">
-        <div className="cart-aside-part">
-          <div className="cart-aside-header">
-            <h3>SHOPPING CART</h3>
-            <button className="x-button cart-aside-x">
-              <span className="sidebar-button-text">
-                <AiOutlineClose className="button-x formobileicon" />
-                <span className="button-title">Close</span>
-              </span>
-            </button>
-          </div>
-          <div className="cart-aside-content">
-            <ul>
-                <li>
-                    <div className="list-img">
-                        <img src={bowl} alt="bowl" />
-                    </div>
-                    <div className="list-content">
-                        <p>Keny Bowl</p>
-                        <p>QTY: 2</p>
-                        <p>$25.00</p>
-                    </div>
-                </li>
-            </ul>
+    <aside className={asideOpener.isCartOpen ? 'cart-aside-field cart-aside-open' : 'cart-aside-field'}>
+      <div className="cart-aside-part">
+        <div className="cart-aside-header">
+          <h3>SHOPPING CART</h3>
+          <button className="x-button cart-aside-x" onClick={closeCartAside}>
+            <span className="sidebar-button-text">
+              <AiOutlineClose className="button-x formobileicon" />
+              <span className="button-title">Close</span>
+            </span>
+          </button>
+        </div>
+        <div className="cart-aside-content">
+          <ul>
+            <li>
```

Comment:
please create a component for such list items

Reply:
oh i was making design that is why i didnt add component sorry i forgot

Reply:
Done.

- Gonna change inside of them after u merge productCard branch